src/main/java/reactor/netty/http/server/HttpServerResponse.java

```diff
@@ -158,6 +159,7 @@
 	 * @param websocketHandler the I/O handler for websocket transport
 	 * @return a {@link Mono} completing when upgrade is confirmed, otherwise fails
 	 */
+	@Deprecated
```

Comment:
Let's keep this method for convenience for those that are OK with the default condifiguration. As you can see in the tests this method is used the most.
At the end we will have two methods
- `sendWebsocket(BiFunction)`
- `sendWebsocket(BiFunction, WebSocketConfigurer)`